### DISCUSSION TITLE
Is there any up-to-date documentation on enabling screen recording for iOS simulators?

### DISCUSSION THREAD

**Discussion by tking16** [appium/appium#19705] [Q&A] [answered]
I had a look [here](https://appium.readthedocs.io/en/stable/en/commands/README/) but the screen recording links seems to be broken

**Comment by KazuCocoa** [comment, marked as answer]
Is this https://appium.github.io/appium-xcuitest-driver/5.14/reference/commands/#startrecordingscreen ?
